generate/src/contract.rs

```diff
@@ -104,6 +106,13 @@ impl Context {
             }
         }
 
+        let event_derives = args
+            .event_derives
+            .iter()
+            .map(|derive| syn::parse_str::<TypePath>(derive))
+            .collect::<Result<Vec<TypePath>, syn::Error>>()
```

Comment:
ubernit: the types can be determined by the compiler:
```suggestion
            .collect::<Result<Vec<_>, _>>()
```
I don't think being explicit here adds value especially since we coerse all errors into `anyhow::Error`s and the `Vec` item type is on the line right above.